packages/core/src/external/commonwell/management/api.ts

```diff
@@ -0,0 +1,147 @@
+import axios from "axios";
+import dayjs from "dayjs";
+import duration from "dayjs/plugin/duration";
+import {
+  Cookie,
+  cookieFromString,
+  CookieManager,
+  cookiesToString,
+} from "../../../domain/auth/cookie-management/cookie-manager";
+import { MetriportError } from "../../../util/error/metriport-error";
+import { safeStringify } from "../../../util/string";
+
+dayjs.extend(duration);
+
+const DEFAULT_TIMEOUT_GET_MEMBER = dayjs.duration({ seconds: 10 });
+const DEFAULT_TIMEOUT_INCLUDE_LIST = dayjs.duration({ seconds: 20 });
+
```

Comment:
timed out when executing this w/ 10s: https://us-east-2.console.aws.amazon.com/cloudwatch/home?region=us-east-2#logsV2:log-groups/log-group/$252Faws$252Flambda$252FCWEnhancedCoverageSessionMgmtLambda/log-events/2023$252F10$252F29$252F$255B$2524LATEST$255D909d66264d5e4e599405fc84bbe7f796$3Fstart$3D1698611455643$26refEventId$3D37880301265829819321062405600970396609830755635652001792